addons/website/tests/test_ui.py

```diff
@@ -155,3 +155,56 @@ def test_06_public_user_editor(self):
 
     def test_07_carousel_snippet_content_removal(self):
         self.start_tour("/", "carousel_content_removal", login='admin')
+
+    def test_08_focus_blur_unit_test(self):
+        # TODO this should definitely not be a website python tour test but
+        # while waiting for a proper web_editor qunit JS test suite for the
+        # editor, it is better than no test at all as this was broken multiple
+        # times already.
+        self.env["ir.ui.view"].create({
```

Comment:
You can create the 3 views in a single create so you don't have to recall the env[iruiview]
(not 100% sure in 13.0 tho)